USER_GUIDE.md

```diff
@@ -266,6 +266,42 @@ DeleteDataStreamRequest deleteDataStreamRequest = new DeleteDataStreamRequest.Bu
 DeleteDataStreamResponse deleteDataStreamResponse = javaClient().indices().deleteDataStream(deleteDataStreamRequest);
 ```
 
+## Point-In-Time API
```

Comment:
This addition will require updating the TOC.

Reply:
Added.